Client.xcodeproj/project.pbxproj

```diff
@@ -2713,10 +2717,7 @@
 			isa = PBXGroup;
 			children = (
 				D32075381B017DA400CA1CD2 /* SearchPlugins */,
-				0BA1E02F1B051A07007675AF /* NetError.css */,
```

Comment:
I had these changes in my working copy as well but I choose to not include it. I think we want to keep these files no?